ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,54 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+Most of the content on [ocf.berkeley.edu](https://ocf.berkeley.edu)
+is created and owned by OCF users, not the OCF itself.
+
+The OCF maintains a strict policy not to remove or modify data
+hosted by members unless legally compelled to do so, or if such
+data is in violation of existing OCF [account policies][policy].
+
+This means the OCF will not change or remove any text, pictures,
+information, or other data hosted by an OCF member. We do however
+reserve the right to deactivate inactive accounts at our discretion.
+
+## Suggestions
+
+If you take issue with the content hosted on the website of an OCF user,
+the responsibility falls on you to resolve any conflict.
+
+We expect you to make a serious attempt to contact the owner of
+the content in question and request that they address your concern
+directly. You may try to find their contact information on their
+website or in the [University directory](//directory.berkeley.edu).
+If this fails, you may try to contact the owner by the contact email
+associated with their OCF account. If the website in question is
+`https://www.ocf.berkeley.edu/~username`, the contact address will be
+<username>@ocf.berkeley.edu.
+
+If repeated attempts to contact the owner of the account have failed,
+you may [contact us](//ocf.io/contact) and request the website be
+disabled due to inactivity. If we attempt to contact the website owner
+and fail to get a response, we may disable their account as per our
+[account policies][policy]
+
+Please note that, if the above steps fail, the website owner is maintaining
+their website in compliance with our policies. There is nothing the OCF can
```

Comment:
> these sentences don't make sense to me. how does them refusing to remove content imply they are in compliance with our policies? they might not be

The point is to discourage people from asking us to take down information if the website owner has turned them down. I can see why this is a problem, though. Perhaps add a short paragraph above along the lines of
> If you get in contact with the website owner and they refuse your removal request, the OCF will not intervene unless the website owner is in violation of our [[account policies|link|]. If you believe part of a website to be in violation of these policies, you may contact us and name the exact policy you think has been breached. If we determine the complaint to be valid, we will order the website owner to remove the offending information. The website will be disabled if its owner fails to comply.